### DISCUSSION TITLE
验证码预填新功能关于周六bw抢票的疑问

### DISCUSSION THREAD

**Discussion by WanWuDaQian** [mikumifa/biliTickerBuy#362] [提问专区]
![截图_20240704004021](https://github.com/mikumifa/biliTickerBuy/assets/159460732/8b4f4829-a4cb-420c-9c49-0a026867fb83)
大佬新功能nb！
有个小小的疑问~
要抢的票仓中其它已售罄或者已开票还未售罄的票种配置文件才能有用，大家可知道这次周六bw该怎么选？
目前bw的票仓里均是“已售罄”状态，但周六bw应该是“未开票”状态，好像均不满足上述条件？“验证码提前预填时间”总不能提前几个小时吧？
本人理解能力实在有限，求大佬们解答。。。┭┮﹏┭┮    ORZ—

**Comment by SouthSeafireworks** [comment]
用联票来刷新验证码，就看到时候联票他刷不刷新了，第二波不开联票，所以理论上来说后面要是有回流票的话就不至于直接给整个联票关了，联票还是不会刷新，处于已售罄状态

**Reply by WanWuDaQian** [reply]
多谢了！！！

**Reply by TNKZC** [reply]
您好，看了讨论我也有点小小的疑惑，我看说明中是其它《已售完》或者《已开票还未售完》的票种，BW的票仓现在均是已售完状态，应该是符合条件的。楼主的意思是说周六时BW的票仓会全部更新为未开票状态，需要新的配置文件，所以不符合条件吗？
理解偏差，还望您解答一下，感谢大佬～

**Reply by gpc123456** [reply]
建议您在抢票前1-2个小时再生成验证码预填的配置文件，如果到时候票仓中仍有`已售罄`状态的票种，就可以使用该票种生成配置文件来预填验证码；如果都显示`不可售`的话，那就无法使用验证码预填了，因为向不可售的票种发送订单准备请求极有可能被B站风控，导致无法抢票。现在的还不太能确定周六BW票仓的配置状况，所以不建议这么早就生成配置文件，至少等到APP上会员购页面中更新开票状态后再生成配置文件。

**Reply by TNKZC** [reply]
明白了，感谢大佬解答！😊
